### ISSUE TITLE
Reviewer 2, Round 2, 8.1 application range of GEMC+TE versus ITIC...

### ISSUE THREAD

**Issue by ramess101** [mostafa-razavi/ITIC-paper#37]
![image](https://user-images.githubusercontent.com/23408516/55338824-b6150280-545e-11e9-8d10-80aab01ca8b6.png)

Since the index for this item is 8.1, the reviewer is referring specifically to the conclusions section (Section 8). We have already compared ITIC with GEMC+TI in the manuscript (and even towards the end of the conclusions) but I believe this is the paragraph the reviewer wants us to mention TI.

![image](https://user-images.githubusercontent.com/23408516/55338445-fc1d9680-545d-11e9-9d5c-12e207fff258.png)

I would modify the text to read:

"due to the insertion/transfer moves, unless combined with thermodynamic integration. [missing period before] The ITIC method, hence, outperforms GEMC and GCMC when Tr is less than 0.6, while spanning a similar temperature range as GEMC+TI (Tr = 0.3)."

**Comment by ramess101** [comment]
@mostafa-razavi 

I just realized that we shouldn't use the terminology "temperature range" because that implies the same upper bound as well. So let's change this to something like "while attaining [or reaching or obtaining] a similar lower temperature value [or bound or limit] as GEMC-TI..."

**Comment by ramess101** [comment]
@mostafa-razavi 

![image](https://user-images.githubusercontent.com/23408516/55349472-0bf5a480-5477-11e9-8ec3-6c95f401df35.png)

Again, I think the comment was mainly aimed at the conclusions. Or the reviewer might not have even noticed that ethane results.

**Comment by mostafa-razavi** [comment]
@ramess101  @jrelliottoh 

I simulated TraPPE-C2 and TraPPE-iC6 in Cassandra at low temperatures using Gibbs-Duhem integration method. In the following plot, the five lowest temperature points of TraPPE-iC6 (unfilled brown circles) and the 5 lowest temperature points of TraPPE-C2 (unfilled cyan diamonds) are Gibbs-Duhem results. I haven't added these to the manuscript, but we should probably do that and say something about it in response to reviewer's "Item 8.1" comment.

![Deviation-Psat-rhoL-rhoV](https://user-images.githubusercontent.com/16358113/55464040-43279b00-55c8-11e9-9b64-1c911092430b.png)

**Comment by jrelliottoh** [comment]
do you have estimates of CPU time comparisons?JRE


    On Wednesday, April 3, 2019, 4:25:09 AM EDT, mostafa-razavi <notifications@github.com> wrote:  
 
 
@ramess101 @jrelliottoh

I simulated TraPPE-C2 and TraPPE-iC6 in Cassandra at low temperatures using Gibbs-Duhem integration method. In the following plot, the five lowest temperature points of TraPPE-iC6 (unfilled brown circles) and the 5 lowest temperature points of TraPPE-C2 (unfilled cyan diamonds) are Gibbs-Duhem results. I haven't added these to the manuscript, but we should probably do that and say something about it in response to reviewer's "Item 8.1" comment.

**Comment by ramess101** [comment]
I couldn’t tell which points were the new ones

**Comment by mostafa-razavi** [comment]
@jrelliottoh 
> do you have estimates of CPU time comparisons?JRE

Yes. For iC6 it takes 10 hours to perform NPT simulation in liquid phase (which is slower) at each temperature. The important issue about Gibbs-Duhem is that you need to go to lower temperatures sequentially. So if one data point takes 10 hours, 5 data points take 50 hours. In other words, you can't run Gibbs-Duhem in parallel. ITIC has an advantage here.

**Comment by ramess101** [comment]
@mostafa-razavi 

> I haven't added these to the manuscript, but we should probably do that and say something about it in response to reviewer's "Item 8.1" comment.

I have two hesitations: 1) Item 8.1 was really just referring to the conclusion section. We already satisfied this reviewer's concern regarding TI in the body of the text. 2) The empty cyan and empty brown symbols are mainly from the literature. It would be inconsistent/confusing to have most of the points be from the literature and just a few of the lower points to be from TI. Also, we never mention in the manuscript that we performed these simulations and we probably need to provide some details so that someone could replicate our results. My vote would be to leave it out of this review.
